cozy.example.yaml

```diff
@@ -318,10 +318,16 @@ contexts:
     init_photos_folder: true
     # Allows to override the default template "Cozy" title by your own title
     templates_title: "My Personal Cloud"
-    # Use a different noreply mail for this context
-    noreply_address: noreply@cozy.beta
-    noreply_name: My Cozy Beta
-    reply_to: support@cozy.beta
+    # Use a different config for mails for this context
+    mail:
+      # If the host is set to "-", no mail will be sent on this context
+      host: smtp.cozy.beta
+      port: 465
+      username: {{.Env.COZY_BETA_MAIL_USERNAME}}
+      password: {{.Env.COZY_BETA_MAIL_PASSWORD}}
```

Comment:
Contexts are publicly accessible so it is not an option to store a password in contexts. please move it outside of context

Reply:
Good catch! I could remove the mail entry from the HTTP response on the public route, but it is probably safer to not store mail configuration here.